target/scripts/start-mailserver.sh

```diff
@@ -170,24 +169,35 @@ function _register_functions() {
 # ? >> Executing all stacks / actual start of DMS
 # ------------------------------------------------------------
 
-_early_supervisor_setup
-_early_variables_setup
+# fresh container?
+if [[ ! -f /CONTAINER_START ]]; then
+  # first container start
```

Comment:
Might be better to communicate the intent more explicitly? I know we all know why, but it may not be obvious to other readers (_and hopeful future maintainers_)

```suggestion
# Ensure DMS only adjusts config files for a new container.
# Container restarts should skip as they retain the modified config.
if [[ ! -f /CONTAINER_START ]]; then
```